ios/FluentUI.Demo/FluentUI.Demo/Demos/ShimmerLinesViewDemoController.swift

```diff
@@ -66,9 +66,7 @@ class ShimmerViewDemoController: DemoController {
 
         container.addArrangedSubview(shimmerViewLabel("A ShimmerLinesView needs no containerview or subviews"))
         container.addArrangedSubview(dividers[0])
-        container.addArrangedSubview(ShimmerLinesView(lineCount: 3,
-                                                         firstLineFillPercent: 0.94,
-                                                         lastLineFillPercent: 0.6))
+        container.addArrangedSubview(ShimmerLinesView())
```

Comment:
why we changing this?

Reply:
I thought it'd be best to keep the shimmer demo consistent both in `main` and `fluent2-tokens`. Should I revert it?

Reply:
so who is right here? main or fluent2-tokens? what cause this divergence?

Reply:
ShimmerLinesView without the parameters is the version from main. I don't know why there's a divergence here.